dali/ft/daftformat.ipp

```diff
@@ -20,9 +20,10 @@
 
 #include "filecopy.hpp"
 #include "ftbase.ipp"
-#include "daft.hpp"
+#include "daftmc.hpp"
```

Comment:
daftmc.hpp defines some log flogs that were necessary for adding LOG statements.  I may move stuff around.